src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -1263,6 +1264,7 @@ void runtime::cpu::CPU_ExternalFunction::register_common_passes(
 
     REGISTER_KNOBBED_PASS(LikeReplacement, true, ngraph::pass)
     REGISTER_KNOBBED_PASS_WITH_ARGS(FusedOpDecomposition, true, ngraph::pass, is_supported)
+    REGISTER_KNOBBED_PASS(Opset1Downgrade, true, ngraph::pass)
```

Comment:
Here we unconditionally convert Opset-3 operations to opser-1. But what if some backend supports opset3 operation? In this case we should not replace it and potentially we loose some performance boost.